packages/typespec-vscode/src/vscode-cmd/install-tsp-compiler.ts

```diff
@@ -0,0 +1,74 @@
+import { cwd } from "process";
```

Comment:
Can you either remove this import (`process.cwd()` is available globally in node.js) or update it to:
```suggestion
import { cwd } from "node:process";
```

Unless you're doing this for web compatibility via the `process` npm package (but I don't see a dependency on it)

Reply:
Updated. From curious, do you mean it will still work if i remove the import? But it would report an error on my side if i remove it...

Reply:
> But it would report an error on my side if i remove it...

Oh I'm surprised by that - `process` is a node.js global. When I removed the import locally it worked fine.

Reply:
i see. I can "." from the process object directly. I prefer this way to the "import". code updated to reomve the import. thanks.